### PR TITLE
Blog post about 1-year announcement anniversary.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@fabxc @brian-brazil @beorn7

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- >200 contributors
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- TODO: PR + commit stats
+- TODO: issue stats
+- TODO: conference talks
+- TODO: articles
```
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: @brian-brazil and @beorn7 - you wanna help out with some of these? Brian, you already mentioned some talk and article stats?

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+## The beginning
+
+Today marks the one-year anniversary of Prometheus being announced to the wider
+world.
+
+Although we already started Prometheus as an open-source project on GitHub in
+2012, we didn't make noise about it at first. We wanted to give the project
+time to mature and be able to experiment without friction. Prometheus was
+gradually introduced for production monitoring at
+[SoundCloud](https://soundcloud.com/stream) in 2013 and then saw more and more
```
Comment (fabxc) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Why the `/stream`?

Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Copy-and-paste bug. Will fix.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 1 above)
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: We're at 68 posts on the subreddit, so I'd presume 100+ articles total given that only covers the past 5 months.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## After the first wave
+
+After those first weeks, the initial onslaught of incoming communication cooled
+down a bit, but we were and still are receiving constantly growing adoption.
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- >200 contributors
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Wasn't it >250?

Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: I didn't count myself, I stole a number from an internal doc from Bjoern or so :) Verification welcome.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- TODO: PR + commit stats
+- TODO: issue stats
+- TODO: conference talks
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: I've given 8 talks so far at meetups/conferences. On top of that we have jaq at monitorama, http://conferences.oreilly.com/oscon/open-source-2015/public/schedule/detail/44789, plus whatever ye've done.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+Both personally and technically, we are really excited about what has happened
+last year in Prometheus-land. We love the opportunity to provide the world with
+a powerful new approach to monitoring, especially one that is much better
+suited towards modern cloud- and container-based infrastructures than
+traditional solutions. We are also very grateful to all contributors and
+hope to continuously improve Prometheus for everyone.
+
+Although Prometheus is relatively mature by now, we have a list of major goals
+we want to tackle in 2016. The highlights will be stabilizing the new
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: This reads as though it isn't already stable (which it as as far as we know), I'd say "polishing" instead.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 4 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: I listed some in my head and ended up with around ~20, so I'm going to go with that.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 5 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Changed to "polishing".

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+someone unknown to us (hello there,
+[jjwiseman](https://news.ycombinator.com/user?id=jjwiseman)!) had submitted
+[the Prometheus website](http://prometheus.io/) to Hacker News and somehow their
+post had made it to [all the way to the top](https://news.ycombinator.com/item?id=8995696).
+
+This is when things started going slightly crazy in a good way. We saw a sharp
+rise in contributors, mailing list questions, GitHub issues, IRC visitors,
+requests for conference and meetup talks, and increasing buzz on the net in
+general. Since the beginning, we have been very lucky about the quality of our
+newly gained community: The kind of people who were attracted to Prometheus
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: s/gained/expanded/ reads a little better

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+worldwide:
+
+[![Prometheus trending on GitHub](/assets/prometheus_github_trending.png)](/assets/prometheus_github_trending.png)
+
+## After the first wave
+
+After those first weeks, the initial onslaught of incoming communication cooled
+down a bit, but we were and still are receiving constantly growing adoption.
+
+To give you an idea of the ecosystem, we now have:
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: 253 people on the mailing list, 330 threads.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- 200+ contributors
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- TODO: PR + commit stats
+- TODO: issue stats
+- ~20 Prometheus-related talks and workshops
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: 20+ sounds better (and there's likely some we don't know about)

Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Done.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 7 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Added.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+down a bit, but we were and still are receiving constantly growing adoption.
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- 200+ contributors
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- TODO: PR + commit stats
+- TODO: issue stats
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: I make it 3546 issues total, of hich 363 are open.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+
+The widely popular dashboard builder [Grafana](http://grafana.org/) also added
+native Prometheus backend support in [version
+2.5](http://grafana.org/blog/2015/10/28/Grafana-2-5-Released.html). Since
+people all around the world are already using and loving Grafana, we are going
+to focus on improving Grafana's Prometheus integration and will likely invest
+less energy in our own dashboard builder
+[PromDash](https://github.com/prometheus/promdash) in the future.
+
+On a funnier note, 2015 has also been the year in which Prometheus's query
```
Comment (brian-brazil) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: s/funnier/lighter/ is more idiomatic.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- 200+ contributors
+- TODO: PR + commit stats
+- TODO: issue stats
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- 253 people on the mailing list and 330 threads
```
Comment (grobie) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: 250+ and 300+ to be more inline with the other numbers?

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 9 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Thanks, I did 3500+ and 350+.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 10 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Done.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 11 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Done.

**Comment by juliusv** [comment]
I also changed the introductory paragraph.

**Comment by grobie** [comment]
Hm, what about mentioning that there is the first professional support company with robust perception? Seems to be an important mile stone to me and having commercial support available is hard requirement for many companies. If @brian-brazil agrees of course.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+down a bit, but we were and still are receiving constantly growing adoption.
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- 200+ contributors
+- TODO: PR + commit stats
+- 3500+ issues in total, 350+ open
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
```
Comment (grobie) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: 150+

**Comment by brian-brazil** [comment]
I agree, I've talked to many companies in that position.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 12 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Sometimes it's less, but ok :) Changing.

**Comment by juliusv** [comment]
Good idea, will add that.

**Comment by juliusv** [comment]
@brian-brazil Check out my latest commit, which adds a Robust Perception mention!

**Comment by brian-brazil** [comment]
:+1:

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+[DreamHack](https://www.dreamhack.se), has [used
+Prometheus](http://prometheus.io/blog/2015/06/24/monitoring-dreamhack/) to keep
+tabs on their network infrastructure in 2015, and
+[FOSDEM](https://fosdem.org/2016/) will do so in 2016.
+
+The widely popular dashboard builder [Grafana](http://grafana.org/) also added
+native Prometheus backend support in [version
+2.5](http://grafana.org/blog/2015/10/28/Grafana-2-5-Released.html). Since
+people all around the world are already using and loving Grafana, we are going
+to focus on improving Grafana's Prometheus integration and will likely invest
```
Comment (grobie) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: s/likely//

Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Removed.

**Comment by beorn7** [comment]
Will have a look tomorrow, and work on TODOs still to be done then.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+the beginning.
+
+Although we already started Prometheus as an open-source project on GitHub in
+2012, we didn't make noise about it at first. We wanted to give the project
+time to mature and be able to experiment without friction. Prometheus was
+gradually introduced for production monitoring at
+[SoundCloud](https://soundcloud.com/) in 2013 and then saw more and more
+usage within the company, as well as some early adoption by our friends at
+Docker and Boexever in 2014. Over the years, Prometheus was growing more and
+more mature and although it was already solving people's monitoring problems,
```
Comment (beorn7) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Would a comma after 'mature' help parsing (to note that the 2nd "and" connects two sentences into one)? (Or break this long complicated sentence into two?)

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+also turned out to be very competent, constructive, and high-quality
+contributors and users. The ideal open-source scenario of receiving a lot of
+value back from the community was a reality pretty much from day one.
+
+What does all that Hacker News buzz look like in terms of GitHub stars? Try and
+see if you can find the exact moment in this graph (ironically, a Gnuplot and
+not Prometheus graph) when we went out of "dark mode" and got hit by Hacker
+News:
+
+[![Prometheus GitHub stars](/assets/prometheus_github_stars.png)](/assets/prometheus_github_stars.png)
```
Comment (beorn7) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Mandatory reference: https://xkcd.com/1179/

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- 200+ contributors
+- TODO: PR + commit stats
+- 3500+ issues in total, 350+ open
+- 150+ people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- 250+ people on the mailing list and 300+ threads
```
Comment (beorn7) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: s/and/created/ ?

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+more. He is currently working on the new beta rewrite of the
+[Alertmanager](https://github.com/prometheus/alertmanager).
+
+Finally, we have been honored to be recognized and adopted by major players in
+the industry. [Google](https://www.google.com) is now instrumenting its open-source
+container management system [Kubernetes](http://kubernetes.io/) natively with
+Prometheus metrics. [CoreOS](https://coreos.com/) is picking it up for
+[etcd](https://coreos.com/etcd/)'s monitoring as well. [DigitalOcean](https://www.digitalocean.com/) is betting on Prometheus for their
+internal monitoring. By now, the list of companies using Prometheus in one way
+or another has become too long to mention: [Google](https://www.google.com),
```
Comment (beorn7) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: To mention what? Every single user?

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+With the Prometheus ecosystem continuing to grow, the first users have started
+asking about commercial support. While Prometheus will always remain an
+independent open source project, one of our core contributors ([Brian
+Brazil](https://github.com/brian-brazil)) has recently founded his own company,
+[Robust Perception](http://www.robustperception.io/), which provides support
+and consulting services around Prometheus and monitoring in general.
+
+On a lighter note, 2015 has also been the year in which Prometheus's query
+language was proven to be Turing complete by Brian by implementing
+[Conway's Game of Life in PromQL](http://www.robustperception.io/conways-life-in-prometheus/).
```
Comment (beorn7) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: On a lighter note, 2015 has also been the year in which Brian proved Prometheus's query language to be Turing complete by implementing ...

(No passive voice, no double "by".)

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 17 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: How about "to mention all of them"? (companies)

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 16 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Done.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 18 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: Done.

File: content/blog/2016-01-26-one-year-of-open-prometheus-development.md (hunk 15 above)
Comment (juliusv) [inline content/blog/2016-01-26-one-year-of-open-prometheus-development.md]: ISO looked a bit crammed, so I went for `Jan 2014` format.

**Comment by juliusv** [comment]
I addressed all comments and updated the statistics section as well (including PRs and issues).

Let's ship this!

**Comment by fabxc** [comment]
:shipit:

**Comment by juliusv** [comment]
Shipping!

**Pull request merged**